ncep/packages/parallelio/package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
parallelio is supported in the builtin list of packages, so this can be removed.